### ISSUE TITLE
Give full list of dependencies in README

### ISSUE THREAD

**Issue by hamishwillee** [Marxlp/pyFlightAnalysis#5]
In section https://github.com/Marxlp/pyFlightAnalysis#run-from-source , rather than the somewhat imprecise list of dependencies, would be great if you could provide something people can copy-paste like:

```
pip install pyqtgraph pyOpenGL pyulog numpy matplotlib
```
If using Python3, then also install PyQt as shown
```
pip install PyQt5
```
> **Note** If using Python 2, instead install PyQt4 manually ....

**Comment by Marxlp** [comment]
Yes, I have updated the README file. But it still need your review to avoid syntax error.

**Comment by hamishwillee** [comment]
> Yes, I have updated the README file. But it still need your review to avoid syntax error.

The changes are not rendering. I have created #6 which fixes this and improves the English a little.

This can be closed if #6 is merged.

**Comment by Marxlp** [comment]
Fine.
